.eslintrc.yml

```diff
@@ -9,22 +9,28 @@ env:
   node: true
 
 rules:
+  arrow-body-style: 2
+  arrow-parens: [2, as-needed]
```

Comment:
I would like this to be always, and here's my reasoning:

We require curly braces around all conditional and loop statements. The reason we do this (and why it's a standard across JS development just now) is because you can try to add a second line to an if, and if you forge tto add in the brackets afterwards, it won't do what you expect (it'll run the second line always).

I feel like we have the same issue with arrow functions, where you can technically do a 1 argument arrow function which doesn't have parenthesis, but if you add in a second argument, and forget to add in the brackets, you will end up with issues. The extra characters don't add much work, and they stop us having issues in the future. Also, it keeps all arrow functions looking consistent.